src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -625,6 +639,14 @@ public String getCaseFormat(UUID caseUuid, UUID studyUuid, String userId) {
         }
     }
 
+    public String getCaseFormat(UUID caseUuid) {
```

Comment:
Method already exists ?

Reply:
method exists with a different signature, I'm not creating a study here
maybe we should refactor "getCaseFormat" original method ?